src/modules/account/account.entity.d.ts

```diff
@@ -33,6 +33,13 @@ export type GetCredentialIdByEmailOutput = Pick<
     >;
   };
 };
+export type GetCredential = Pick<Credential, 'id' | 'email'> & {
+  parentProfile: Pick<ParentProfile, 'id' | 'fullname'> & {
```

Comment:
Não retornar id da credencial, nem dos pais. esse id é obtido pela chave de acesso

```suggestion
export type GetCredential = Pick<Credential, 'email'> & {
  parentProfile: Pick<ParentProfile, 'fullname'> & {
```